src/top/mod.rs

```diff
@@ -20,6 +20,6 @@ pub struct Opts {
     url: Option<Url>,
 
     /// Humanize metrics, using numeric suffixes - e.g. 1,100 = 1.10 k, 1,000,000 = 1.00 M
-    #[clap(short, long)]
+    #[clap(visible_alias = "hm", long)]
```

Comment:
I'd suggest using `-H`.

Reply:
Sounds good, I prefer that since the alias isn't a true short flag.

Reply:
Oh that's much better. Jesse is so smart